validator/slashing-protection-history/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package slashingprotection defines methods to parse, import, and export slashing protection data
+// from a standard JSON file according to EIP-3076 https://eips.ethereum.org/EIPS/eip-3076. This format
+// is critical to allow safe interoperability between Ethereum consensus clients.
+package slashingprotectionhistory
```

Comment:
Thanks for making package comment files, seeing this is a good reminder for me to do more of that. I used to have a practice of always running godoc to browse how my final docs looked and think about whether I could learn to use a package from that viewpoint, but I've gotten way too lazy about that!